src/components/shared/sidebar/Sidebar.tsx

```diff
@@ -5,6 +5,8 @@ import Link from "next/link";
 import { Github, LucideLinkedin } from "lucide-react";
 import { ModeToggle } from "../../theme/Toggle";
 import ProfileAdd from "./ProfileAdd";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faDiscord } from "@fortawesome/free-brands-svg-icons";
```

Comment:
Please Keep using **[lucide-react](https://lucide.dev/icons/)**. I don't think we need to use any other `icon` library for only **discord** icon.

Reply:
Okay, I will do these by tonight or early tomorrow morning if that's okay with you?

Reply:
Thanks for your response,🤩
no problem whenever you are free you can do it 🙃